cluster/log-dump.sh

```diff
@@ -133,13 +134,16 @@ function save-logs() {
           ;;
       esac
     fi
+    local -r services=( ${systemd_services} ${LOG_DUMP_SAVE_SERVICES:-} )
```

Comment:
It does, because the original variables are using the "space separated list" style. If you do e.g.
```
foo="a b"
bar="c"
arr=( "${foo}" "${bar}" )
echo ${arr[0]}
```

... you'll see "a b", which makes a certain perverse sense.

The weirder one is quoted array expansion, which requotes each element, so e.g. this copies the array, even with one element containing a space:
```
arr=( "a b" "c" )
arr2=( "${arr[@]}" ) # quotes, this expands to: "a b" "c"
```